python/src/opendp/_extrinsics/_make_private_selection/__init__.py

```diff
@@ -0,0 +1,86 @@
+from typing import Optional
+
+import opendp.prelude as dp
+from opendp._lib import import_optional_dependency
+from opendp.mod import Measurement
+
+
+def make_private_selection_threshold(meas: Measurement,
+                                     threshold: float,
+                                     stop_probability: float,
+                                     epsilon_selection: float,
+                                     steps: Optional[int] = None):
+
+    """Measurement for private selection with known threshold.
+
+    This executes an ε-DP mechanism M that returns a tuple (q, x) repeatedly
+    until the score q is at least T, a predefined threshold.
+    Only the last output of M is released.
+
+    Algorithm 1: https://arxiv.org/pdf/1811.07971.pdf#page=7
+
+    :param meas: A measurement function that returns a 2-tuple when invoked,
+                 where the first element is a score.
+    :param threshold: The threshold score. Return immediately if the score is above this threshold.
+    :param stop_probability: The probability of stopping early at any iteration.
+    :param epsilon_selection: The epsilon allocated to the private selection.
+    :param steps: Optional. The number of steps to run. If not specified, will run the minimum number of steps.
```

Comment:
Are `stop_probability` and `steps` about exiting early, before a solution is found? Or something else? Might also mention the particular exception is that is raised on an early exit.

Reply:
(see reply below)